interface/package.json

```diff
@@ -14,7 +14,9 @@
   "engines": {
     "node": ">=4.0"
   },
-  "dependencies": {},
+  "dependencies": {
+    "hclusterjs": "^1.3.0"
```

Comment:
I believe this is the first NPM module we are deploying into our front-end code rather than using as a tool operating in our build process. I think this means it belongs in `dependencies` rather than `dev-dependencies`.

Would there be some benefit from changing things around to have it appear in `vendors/` instead?